examples/chart/teleport-cluster/templates/auth/deployment.yaml

```
@@ -212,6 +212,13 @@ spec:
       - name: "operator"
         image: '{{ .Values.operator.image }}:{{ include "teleport-cluster.version" . }}'
         imagePullPolicy: {{ .Values.imagePullPolicy }}
+        {{- if or $auth.extraEnv $auth.tls.existingCASecretName }}
+        env:
+        {{- if $auth.tls.existingCASecretName }}
+        - name: SSL_CERT_FILE
+          value: /etc/teleport-tls-ca/ca.pem
+        {{- end }}
+        {{- end }}
```

No need to check for the `extraEnv` as we don't set it.

nit: the gotemplate is indented based on the `if`-level in this file

```suggestion
  {{- if $auth.tls.existingCASecretName }}
        env:
        - name: SSL_CERT_FILE
          value: /etc/teleport-tls-ca/ca.pem
  {{- end }}
```